pyhive/presto.py

```diff
@@ -80,7 +80,7 @@ class Cursor(common.DBAPICursor):
     visible by other cursors or connections.
     """
 
-    def __init__(self, host, port='8080', username=None, catalog='hive',
+    def __init__(self, host, port='8080', username=None, principle_username=None, catalog='hive',
```

Comment:
could you please provide a bit more context, maybe in the comment on differences between username & principle_username and share more information why username cannot be used ?

Reply:
Comment added detailing the difference between the two fields and also added a documentation link to the feature in question.